### PR TITLE
TOTD: Exchange Gitter for Discord, Fix date format

### PR DESCRIPTION
**Pull request by tooomm**
## Short roundup of the initial problem
Gitter is no longer used.
Date format YYYY.MM.DD does not even exist.

## What will change with this Pull Request?
- Add Discord community
- Picture taken from https://discord.com/branding
- Use international date format (screenshot has old one still)

## Screenshots

image

### CODE REVIEW

File: cockatrice/src/dlg_tip_of_the_day.cpp [hunk 1]
```diff
@@ -154,7 +154,7 @@ void DlgTipOfTheDay::updateTip(int tipId)
         imageLabel->setPixmap(image->scaled(w, h, Qt::KeepAspectRatio, Qt::SmoothTransformation));
     }
 
-    date->setText("<i>Tip added on: " + tip.getDate().toString("yyyy.MM.dd") + "</i>");
+    date->setText("<i>Tip added on: " + tip.getDate().toString("yyyy-MM-dd") + "</i>");
```
[the remark this answers is not in the document]
Comment (tooomm) [reply cockatrice/src/dlg_tip_of_the_day.cpp:157]: Not sure what you are trying to tell me here :)
https://en.wikipedia.org/wiki/Date_format_by_country

What's wrong with using ISO 8601?

[the remark this answers is not in the document]
Comment (ebbit1q) [reply cockatrice/src/dlg_tip_of_the_day.cpp:157]: it's fine, just wondering why this was important to change

**Review by ebbit1q — APPROVED** [review]
thanks for noticing this tip was outdated, there might be more tips that could be changed

File: cockatrice/src/dlg_tip_of_the_day.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (tooomm) [reply cockatrice/src/dlg_tip_of_the_day.cpp:157]: Not essential or even important, but an very easy and quick fix "for free" when passing by and replacing the outdated tip anyway. :)

**Pull request merged**